packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -438,6 +448,7 @@ export const ChatScreen = () => {
               styles.keyboardAvoiding,
               hasCurrentlyPlayingTrack ? { bottom: PLAY_BAR_HEIGHT } : null
             ]}
+            onKeyboardHide={() => measureView(composeRef, chatContainerBottom)}
```

Comment:
This is going to change on every render. Might be worth pulling it out into a `useCallback` if `composeRef` and `chatContainerBottom` don't change on every render.